src/index.ts

```diff
@@ -1,2 +1,4 @@
 export * as Clients from './clients';
+export * as Encoding from './encoding';
+export * as StorageProofs from './storage-proofs';
```

Comment:
We should use uppercase on first letter when its a class otherwise casing should be like this
```suggestion
export * as clients from './clients';
export * as encoding from './encoding';
export * as storageProofs from './storage-proofs';
```